fastlane.gemspec

```
@@ -97,7 +97,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency('google-api-client', '>= 0.21.2', '< 0.24.0') # Google API Client to access Play Publishing API
   spec.add_dependency('google-cloud-storage', '>= 1.15.0', '< 2.0.0') # Access Google Cloud Storage for match
 
-  spec.add_dependency('emoji_regex', '~> 0.1') # Used to scan for Emoji in the changelog
+  spec.add_dependency('emoji_regex', '~> 1.0') # Used to scan for Emoji in the changelog
```

```suggestion
  spec.add_dependency('emoji_regex', '>= 0.1' , '< 2.0') # Used to scan for Emoji in the changelog
```

Can we actually change it to be a `>=` and `<=`? This gives us more control over the versions in the future and won't force anybody to bump their `emoji_regex` versions (run `bundle install`) if they aren't even using these feature in pilot 🙃